frontend/src/components/QuestionWrapper.vue

```diff
@@ -0,0 +1,166 @@
+<script setup lang="ts">
+import { computed, defineAsyncComponent } from 'vue';
+import type { ComputedRef, Component } from 'vue';
+import type { Quizoot } from '@schemas/interface';
+import NavigationButton from '@/components/NavigationButton.vue';
+import { pascalToSnake } from '@/lib/string-utils';
+
+interface QuestionsFlow {
```

Comment:
I'm not too keen on this approach. We could merge it into the main interface right? 

We can leave it as is for the time being

Reply:
You're right, it's better to merge it with the main interface